package.json

```diff
@@ -15,6 +15,7 @@
   },
   "devDependencies": {
     "dotenv": "^4.0.0",
+    "pm2": "^3.1.3",
```

Comment:
Do we need this when we install pm2 globally in the `Dockerfile`?

Reply:
True. I had just installed it as a dev dependency for testing, before I got to modifying the `Dockerfile`. I can remove it though.

Reply:
I'd probably just remove it. Unless you plan to make an npm script that runs it 😃

Reply:
done.